assets/scripts/script.js

```diff
@@ -122,6 +122,10 @@
         $thumbnailSliderEl.get(0).style.setProperty('--thumbnail-count', noCols);
       }
 
+      // If less thumbs than slider viewport, don't show prev/next arrows.
+      const numberOfThumbnails = $thumbnailSliderEl.find('li').length;
+      thumbnailSliderArgs.prevNextButtons = numberOfThumbnails > noCols ? true : false,
```

Comment:
I think you can get away with just the conditional cause it will return a boolean.
```suggestion
      thumbnailSliderArgs.prevNextButtons = numberOfThumbnails > noCols;
```